parsec/core/fs/workspacefs/workspacefile.py

```diff
@@ -49,7 +49,7 @@ class WorkspaceFile:
     mode("r") -> read file.
     """
 
-    def __init__(self, transactions, path, mode: str = "r"):
+    def __init__(self, transactions, path, mode: str = "rb"):
```

Comment:
The docstring about the `b` mode seems kind of weird: `bytes, have to be used with another mode, ignored, class is already working with bytes.`
Why are we saying it is ignored when checks are done in the write method that depend on this param ?

Reply:
@Sithi5 Could you fix the docstring?

Reply:
done